### ISSUE TITLE
 An error has been caught in some process when try download some course

### ISSUE THREAD

**Issue by All3yp** [Anon-Exploiter/ine-dl#23]
Hi :)
Always when I try to download a course, (`python .\ine.py -c <id_course_here>`) the output show me this on the print OBS: I have a paid account

![image](https://github.com/user-attachments/assets/a7ea8255-8dbf-46de-a199-f10988ec6868)

**Comment by Anon-Exploiter** [comment]
Hi,

Can you try running this in WSL or Linux machine? Windows terminal is having issues with creating the directory with that name.

**Comment by Anon-Exploiter** [comment]
As stated here https://github.com/Anon-Exploiter/ine-dl?tab=readme-ov-file#works-on

**Comment by All3yp** [comment]
Thank you guys, I solved this part using wsl, I didn´t see this part on readme

But I have another problem here:


| When I tried download  it         | When I list courses by category (Cyber security here)       |
|-----------------------------|-----------------------------|
| ![Captura de tela 2024-12-12 181329](https://github.com/user-attachments/assets/7445cec9-a875-4976-8456-6c121a754da6) |![Captura de tela 2024-12-12 180705](https://github.com/user-attachments/assets/6e5efafe-9f43-44f0-b00c-a58f679667bb) |

**Comment by Anon-Exploiter** [comment]
Do you have access to the courses in web app? Can you confirm?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by All3yp** [comment]
I solved the problem, thank you for the help. It was because I only have access to the EJPT path to study. I tried downloading 'Assessment Methodologies: Information Gathering' again, and it worked.

Btw I was wondering why I couldn't download [the entire EJPT path](https://my.ine.com/CyberSecurity/learning-paths/61f88d91-79ff-4d8f-af68-873883dbbd8c/penetration-testing-student), and I didn't find it on the list. But I think it's because I actually need to download each course individually
